INSTALL.md

```diff
@@ -17,6 +19,33 @@ Table of Contents:
 
 6. [More SCons options](#6-more-scons-options)
 
+0. Overall summary
+==================
+
+While the sections below detail how to install GalSim including its required and
+optional dependencies, this section gives a brief summary.  A minimal
+installation of GalSim requires Python (versions 2.6, 2.7, 3.4, or 4.5) along
+with the following dependencies: (a) SCons (versions 2.0 or 2.1 recommended),
+(b) NumPy (version 1.11 recommended), (c) astropy (version 1.1.1 recommended),
+(d) the future module (version 0.16.0 recommended), (e) FFTW (version >=3, often
+called fftw3), (f) TMV (version >=0.72), and (g) boost (version 1.61
+recommended).
+
+A few optional dependencies provide additional functionality, but GalSim can
+otherwise be compiled and used without them.  To get WCS functionality with
+astropy, scipy is required (since it is a dependency of astropy WCS tools).  To
```

Comment:
For anyone who cares about good WCS functionality, I'd definitely recommend starlink, rather than astropy.  It's really far superior, so if we say anything about WCS here, I think that should be the one to mention.